v1/gerDe.json

```diff
@@ -82,15 +82,15 @@
     },
     {
       "id": "number_death",
-      "value": "206"
+      "value": "253"
     },
     {
       "id": "number_active_cases_label",
       "value": "Aktive Coronavirus-Fälle in Deutschland "
```

Comment:
Sollen wir auf _**Aktive Covid-19-Fälle in Deutschland**_ ändern? Wäre bzgl. Kommunikation einheitlicher - siehe Website.

Reply:
Es sind eh eigentlich keine *aktiven* Fälle, weil das RKI die nicht published so weit ich das sehen kann.